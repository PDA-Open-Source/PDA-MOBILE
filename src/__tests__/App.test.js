import React from 'react';
import renderer from 'react-test-renderer';
import App from '..';


jest.mock('react-native-device-info');

describe('App Compontent',()=>{
  it('redner app compontent',()=>{
    const snap = renderer.create(<App/>);
    expect(snap.toJSON()).toMatchSnapshot();
  });
});