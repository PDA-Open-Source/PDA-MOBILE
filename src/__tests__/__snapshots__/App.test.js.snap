// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Compontent redner app compontent 1`] = `
Array [
  <View
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Array [
          Object {
            "flex": 1,
            "flexDirection": "column-reverse",
            "overflow": "hidden",
          },
          Object {
            "backgroundColor": "#000",
          },
        ]
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          accessibilityElementsHidden={false}
          pointerEvents="auto"
          style={
            Object {
              "backgroundColor": "#E9E9EF",
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "paddingTop": 0,
              "position": "absolute",
              "right": 0,
              "shadowColor": "black",
              "shadowOffset": Object {
                "height": 0,
                "width": 0,
              },
              "shadowOpacity": 0.2,
              "shadowRadius": 5,
              "top": 0,
              "transform": Array [
                Object {
                  "translateX": 0,
                },
                Object {
                  "translateY": 0,
                },
              ],
            }
          }
        >
          <View
            accessibilityIgnoresInvertColors={true}
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <Image
              source={
                Object {
                  "testUri": "../../../src/assets/Images/background.jpeg",
                }
              }
              style={
                Array [
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {
                    "height": undefined,
                    "width": undefined,
                  },
                  undefined,
                ]
              }
            />
            <View
              style={
                Object {
                  "backgroundColor": "rgba(0, 0, 0, 0.6)",
                  "flex": 1,
                  "padding": 39.23529411764706,
                }
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "flex": 1,
                    "justifyContent": "center",
                    "padding": 39.23529411764706,
                  }
                }
              >
                <Image
                  source={
                    Object {
                      "testUri": "../../../src/assets/Images/logo.png",
                    }
                  }
                  style={
                    Object {
                      "height": 282.49411764705883,
                      "width": 282.49411764705883,
                    }
                  }
                  testID="logo"
                />
              </View>
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "bottom": 50,
                      "justifyContent": "center",
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                    },
                  ]
                }
              >
                <RNSpinkit
                  color={4294967295}
                  size={25}
                  style={
                    Array [
                      Object {
                        "height": 25,
                        "width": 25,
                      },
                      undefined,
                    ]
                  }
                  type="Circle"
                />
              </View>
              <Text
                style={
                  Object {
                    "bottom": 0,
                    "color": "white",
                    "marginBottom": 6,
                    "marginRight": 5,
                    "position": "absolute",
                    "right": 10,
                    "textAlign": "right",
                  }
                }
              >
                Version 
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        pointerEvents="box-none"
        style={
          Object {
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFF",
              "transform": Array [
                Object {
                  "translateX": -750,
                },
              ],
            }
          }
        >
          <View
            onLayout={[Function]}
            pointerEvents="box-none"
            style={
              Object {
                "backgroundColor": "#FFF",
                "borderBottomColor": "#A7A7AA",
                "borderBottomWidth": 0.5,
                "height": 64,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            />
          </View>
        </View>
      </View>
    </View>
  </View>,
  <View
    style={
      Array [
        Object {
          "backgroundColor": "red",
          "paddingBottom": 20,
        },
      ]
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "padding": 10,
          "textAlign": "center",
          "transform": Array [
            Object {
              "translateX": 0,
            },
          ],
        }
      }
    >
      You are not connected to Internet
    </Text>
  </View>,
  <View
    style={
      Object {
        "left": 0,
        "opacity": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "transform": Array [
          Object {
            "translateY": -48,
          },
        ],
      }
    }
  />,
]
`;
